openedx/core/djangoapps/credentials/tests/mixins.py

```diff
@@ -38,14 +38,14 @@ class CredentialsDataMixin(object):
                 id=1,
                 username='test',
                 credential=factories.ProgramCredential(
-                    program_id=1
+                    program_uuid='aa7a5470-6962-4ff1-935b-b75f33d977b0'
```

Comment:
These data mixins were a bad idea. Once a bunch of tests start using them, making assumptions about their structure, it becomes almost impossible to change these without breaking tests. In a word, this mixin becomes ossified. Instead of making this change to patch this up, what would it take to replace use of this mixin in tests with calls to the factories local to each test class?